packages/realm-react/src/useQuery.tsx

```diff
@@ -31,6 +31,8 @@ export type QueryHookOptions<T> = {
   keyPaths?: string | string[];
 };
 
+export type QuaryHookPartialOptions<T> = Omit<QueryHookOptions<T>, "query">;
```

Comment:
A small typo:
```suggestion
export type QueryHookPartialOptions<T> = Omit<QueryHookOptions<T>, "query">;
```

I wonder if it would be easier to declare a base type (without the `query`) and declare `QueryHookOptions<T>` as a union with that base type and `{ query: ... }`?